src/restore.js

```diff
@@ -24,35 +24,32 @@ async function run() {
 
     const primaryKey = utils.generateCacheKey(core.getInput('package-manager'), workingDirectory)
     core.saveState(State.CachePrimaryKey, primaryKey)
-
+    let cacheKey
     try {
-      const cacheKey = await cache.restoreCache(cachePaths, primaryKey, RestoreKeys)
+      cacheKey = await cache.restoreCache(cachePaths, primaryKey, RestoreKeys)
       if (!cacheKey) {
```

Comment:
Would be good to restructure it, as this part is only logging things, that are not exactly correct anymore. I would assume that missing cacheKey would fail the setCacheState?

Reply:
done